src/components/forms/badge-form.js

```diff
@@ -326,13 +331,13 @@ class BadgeForm extends React.Component {
           </div>
         </div>
         <div className="row form-group">
-          <div className="col-md-12">
+          <div className="col-md-1TWO">
             <label> {T.translate("edit_ticket.access_levels")}:&nbsp;</label>
             {access_levels}
           </div>
         </div>
-        {badgeType.access_levels.some((al) =>
-          al.name.includes("IN_PERSON")
+        {currentSummit.badge_access_level_types.some((al) =>
```

Comment:
this is not correct
u need to check for the badgeType.access_levels
check[ ](https://github.com/fntechgit/summit-admin/pull/470/files#r1761566835)